draft-ietf-rats-corim.md

```diff
@@ -1772,6 +1778,8 @@ Environment-Claims Tuples (ECT) have five attributes:
 
 * Profile : The profile that defines this tuple. If no profile is used, this attribute is omitted.
 
+* Domain Identifier : The Domain to which a given ECT belongs.
```

Comment:
Why can an environment only belong to one domain? What if I need to change a domain-id naming convention and keep both the old and new around for migration's sake?

It seems like you could have a lot of superimposed topologies that are acceptable, such that a single identifier is too restrictive.

Reply:
We have to start with only one domain, to begin with...

A new DM triple should over-ride an existing domain concept.!